starkinfra/issuingbillinginvoice/__issuingbillinginvoice.py

```diff
@@ -0,0 +1,92 @@
+
+from ..utils import rest
+from starkcore.utils.resource import Resource
+from starkcore.utils.checks import check_date
+
+
+class IssuingBillingInvoice(Resource):
+    """# Billing Transaction object
+        Check out our API Documentation at https://starkbank.com/docs/api#issuing-billing-invoice
+        """
+
+    def __init__(self, id=None, tax_id=None, name=None, fine=None, interest=None, status=None, amount=None,nominal_amount=None,  brcode=None, link=None,  due=None, start=None,end=None, created=None, updated=None):
+        Resource.__init__(self, id=id)
+
+        self._id = id
+        self.tax_id = tax_id
+        self.name = name
+        self.fine = fine
+        self.interest = interest
+        self.status = status
+        self.amount = amount
+        self.nominal_amount = nominal_amount
+        self.brcode = brcode
+        self.link = link
+        self.due = due
+        self.start = start
+        self.end = end
+        self.created = created
+        self.updated = updated
+
+_resource = {"class": IssuingBillingInvoice, "name": "IssuingBillingInvoice"}
+
```

Comment:
Quando está na indentação zero, sempre que for pular uma linha, pula duas
```suggestion


_resource = {"class": IssuingBillingInvoice, "name": "IssuingBillingInvoice"}


```

Reply:
Faz a mesma coisa entre as quebras de linha entre os métodos